.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.8</version>
+    <version>1.13</version>
```

Comment:
though going forward it would be better to use https://github.com/jenkins-infra/github-reusable-workflows/pull/48